File: SDKs/plaid.com/2020-09-14_1.20.6/java/lib/src/main/java/openapisdk/models/shared/IncomeVerificationRefreshResponse.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * IncomeVerificationRefreshResponse
 * IncomeVerificationRequestResponse defines the response schema for `/income/verification/refresh`
**/
public class IncomeVerificationRefreshResponse {
    @JsonProperty("request_id")
    public String requestId;
    public IncomeVerificationRefreshResponse withRequestId(String requestId) {
        this.requestId = requestId;
        return this;
    }
    @JsonProperty("verification_refresh_status")
    public VerificationRefreshStatusEnum verificationRefreshStatus;
    public IncomeVerificationRefreshResponse withVerificationRefreshStatus(VerificationRefreshStatusEnum verificationRefreshStatus) {
        this.verificationRefreshStatus = verificationRefreshStatus;
        return this;
    }
}

File: SDKs/plaid.com/2020-09-14_1.20.6/python/src/sdk/models/shared/incomeverificationrefreshresponse.py
import dataclasses
from enum import Enum
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import verificationrefreshstatus_enum as shared_verificationrefreshstatus_enum


@dataclass_json
@dataclasses.dataclass
class IncomeVerificationRefreshResponse:
    r"""IncomeVerificationRefreshResponse
    IncomeVerificationRequestResponse defines the response schema for `/income/verification/refresh`
    """
    
    request_id: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('request_id') }})
    verification_refresh_status: shared_verificationrefreshstatus_enum.VerificationRefreshStatusEnum = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('verification_refresh_status') }})
    
